app/assets/javascripts/night_mode.js

```diff
@@ -0,0 +1,18 @@
+function set_mode(night_mode)
+{
+    mode = night_mode ? 'dark' :'light'
+    Cookies.set("night_mode", night_mode, { expires: 365 })
```

Comment:
La LocalStorage API è nativa e implementa un'interfaccia molto simile, con il bonus che non dobbiamo gestire e deliverare una libreria in più (sia pure da 1 KB e qualcosa). Le caniuse tables sono praticamente identiche a quelle delle CSS variables che già usiamo ([1](https://caniuse.com/#search=css%20var), [2](https://caniuse.com/#search=localstorage)), per cui proporrei di usare semplicemente quella.

Reply:
@riggraz